src/sentry_plugins/pagerduty/plugin.py

```diff
@@ -49,7 +49,7 @@ def notify_users(self, group, event, fail_silently=False, **kwargs):
         if not self.is_configured(group.project):
             return
 
-        description = event.get_legacy_message()[:1024]
+        description = event.message[:1024]
```

Comment:
Sure this shouldn't rather be `event.title`? `event.message` I would assume is always empty.

Reply:
@mitsuhiko I didn't realise that `event.message` is empty, updating to use title. Should this property still exist then or can it be deleted? https://github.com/getsentry/sentry/blob/master/src/sentry/models/event.py#L539-L543

Reply:
@lynnagara the legacy message did something different. We mostly phased it out now. `event.message` is only filled for event types of type log message. `event.title` returns something for all event types.

Reply:
Should we also exclude message from the public schema in discover if it is deprecated/not working well?